### PR TITLE
Consistency pass of settings pages

### PR DESCRIPTION
**Pull request by Solanya**
- Repeating "settings" on the tab names made them crop when the section title already indicates that these are "Settings". Keys in all languages we could somewhat interpret where changed to remove this.
- As an added thing, "Frames settings" was split into "Toolbar" and "Target Frame".

Two things worth discussing:
- There is the small matter of the map button settings. I moved it in General for now as this was the simplest solution. It's possible it should be in the Directory category instead next to the Location settings, but that requires a lot more changes, so I'd like to double-check if we have an alternative idea first.
- Register was renamed to Directory, which means the non-special categories are not in alphabetical order anymore. Then again, other languages were also not in alphabetical order. Do we still want to change it?

image

### CODE REVIEW

**Review by Meorawr — APPROVED** [review]
One comment that "Chat frames" is used as the string in enUS but a bunch of others are still "Chat".

**Comment by Solanya** [comment]
Put the name singular and reused the global string from Blizzard. Should be good now.

File: totalRP3/Locales/frFR.lua [hunk 1]
```diff
@@ -224,7 +224,7 @@ Gardez à l'esprit que changer ces paramètres peut altérer votre expérience a
 	["CO_ANCHOR_TOP"] = "Haut",
 	["CO_ANCHOR_TOP_LEFT"] = "Haut-gauche",
 	["CO_ANCHOR_TOP_RIGHT"] = "Haut-droite",
-	["CO_CHAT"] = "Discussions",
+	["CO_CHAT"] = "	Fenêtre de discussion",
```
Comment (Meorawr) [inline totalRP3/Locales/frFR.lua:227]: Why the leading space bro

**Review by Meorawr — CHANGES_REQUESTED** [review]

**Review by Meorawr — APPROVED** [review]
I approve of this ramen.

**Pull request merged**